wallet-new/server/Cardano/Wallet/API/V1/Handlers/Addresses.hs

```diff
@@ -8,26 +8,28 @@ import qualified Cardano.Wallet.API.V1.Addresses as Addresses
 import           Cardano.Wallet.API.V1.Types
 
 import           Servant
+import           Test.QuickCheck (arbitrary, generate, vectorOf)
 
 handlers :: Server Addresses.API
 handlers =  listAddresses
        :<|> newAddress
 
 listAddresses :: PaginationParams
               -> Handler (OneOf [Address] (ExtendedResponse [Address]))
-listAddresses PaginationParams {..} =
+listAddresses PaginationParams {..} = do
+  addresses <- liftIO $ generate (vectorOf 2 arbitrary)
```

Comment:
I remember how you asked me to send you my `.dir-locals.el` which sets up 4-spaces indentation. Apparently you forgot to put it to use :wink:

Reply:
@flyingleafe Proof! It's here! 😀 

![screen shot 2017-11-21 at 12 12 29](https://user-images.githubusercontent.com/29383371/33069616-659c285e-ceb5-11e7-90d1-454ad3c75391.png)

I don't know what happened here! Maybe the editor is required my to press <tab> to indent correctly and is not forcing the indentation from first principles? I will have to investigate.

Reply:
@adinapoli-iohk 🤔 
Check out the value of `enable-dir-local-variables` variable in your emacs, it should be set to `t` for dir-locals to work. If it is set to `t`, then I don't have other immediate ideas on why this doesn't work for you.
Anyway, you got my general point - let's indent with 4 spaces, like in the rest of the project :)

Reply:
Aye, it's definitely picking the file up as `stylish-haskell` correctly format on save. No clue! Will investigate. But for the sake of indentation absolutely, leave it with me.